pages/repos.js

```diff
@@ -23,7 +46,21 @@ export default function Repos({ repos }) {
 
       <Page>
         <h1 className="text-4xl mb-4 font-bold">Community Repos</h1>
-
+        <div className=" flex justify-end">
+          <Select
+            name="event-type"
+            value={router.query.sortBy || "pushed-date"}
+            label="Sort by"
+            onChange={(e) =>
+              router.push(`/repos?sortBy=${e.currentTarget.value}`)
+            }
```

Comment:
I have removed the ```{}``` did that on same commit.

Reply:
Sorry my mistake 🤦‍♂️